fotoapparat/src/main/java/io/fotoapparat/hardware/v1/Camera1.java

```diff
@@ -433,4 +443,26 @@ private void recordMethod() {
         );
     }
 
+    @Override
+    public void setFocusArea(Rect cameraViewRect, float x, float y) {
```

Comment:
After reading through this method I think it should be moved out of `Camera1` into routine class as much as possible. Why - because this way it could be tested. In the end, it looks like we don't really need `setFocusArea` in `CameraDevice` itself as we can just update camera parameters from routine class.